JSON/JSON-EVERYTHING_schema.json

```diff
@@ -0,0 +1,469 @@
+            "title": "End Time",
+            "default": "",
+            "type": "string"
+        },
+        "test_system": {
+            "title": "Test System",
+            "default": "",
+            "type": "string"
+        },
+        "score": {
+            "title": "Score",
+            "default": 0.0,
+            "type": "number"
+        },
+        "score_max": {
+            "title": "Score Max",
+            "default": 0.0,
+            "type": "number"
+        },
+        "rules": {
+            "title": "Rules",
+            "oneOf": [
+                {
+                    "type": "object",
+                    "patternProperties": {
+                        "xccdf_[^_]+_rule_.+": {
+                            "$ref": "#/definitions/Rule"
+                        }
+                    }
+                },
+                {
+                    "type": "null"
+                }
+            ]
+        },
+        "groups": {
+            "title": "Groups",
+            "type": "object",
+            "patternProperties": {
+                "xccdf_[^_]+_group_.+": {
+                    "$ref": "#/definitions/Group"
+                }
+            }
+        }
+    },
+    "definitions": {
+        "Identifier": {
+            "title": "Identifier",
+            "type": "object",
+            "properties": {
+                "system": {
+                    "title": "System",
+                    "type": "string"
+                },
+                "text": {
+                    "title": "Text",
+                    "type": "string"
+                }
+            }
+        },
+        "Reference": {
+            "title": "Reference",
+            "type": "object",
+            "properties": {
+                "href": {
+                    "title": "Href",
+                    "type": "string"
+                },
+                "text": {
+                    "title": "Text",
+                    "type": "string"
+                }
+            }
+        },
+        "Remediation": {
+            "title": "Remediation",
+            "type": "object",
+            "properties": {
+                "remediation_id": {
+                    "title": "Remediation ID",
+                    "type": "string"
+                },
+                "system": {
+                    "title": "System",
+                    "default": "",
+                    "type": "string"
+                },
+                "complexity": {
+                    "title": "Complexity",
+                    "default": "",
+                    "type": "string"
+                },
+                "disruption": {
+                    "title": "Disruption",
+                    "default": "",
+                    "type": "string"
+                },
+                "strategy": {
+                    "title": "Strategy",
+                    "default": "",
+                    "type": "string"
+                },
+                "fix": {
```

Comment:
I am not familiar to the JSON standard in details and didn't have opportunity to take a look in the openscap-report, but I believe an remediation which potentially contains json in its content, by any reason, would not cause problems here, right? I don't have an example of this happening in the content, but just exercising the threat modeling here.

Reply:
It doesn't cause a problem. The content of the fix is escaped in this context.